packages/forms/resources/views/components/text-input.blade.php

```diff
@@ -31,7 +31,7 @@
         @endif
 
         @if ($label = $getPrefixLabel())
-            <span @class($affixLabelClasses)>
+            <span @class(array_merge($affixLabelClasses , ['inline-flex items-center rounded-l-lg border border-r-0 border-gray-300 bg-gray-50 px-3 text-gray-500']))>
```

Comment:
Could you add the common affix label classes to `$affixLabelClasses`?

Reply:
Oh yeah totally my bad will do this right now